docs/reference/commandline/volume_ls.md

```diff
@@ -14,14 +14,13 @@ parent = "smn_cli"
 
     List volumes
 
-      -f, --filter=[]      Provide filter values (i.e. 'dangling=true')
+      -f, --filter=[]      Filter output based on these conditions:
+                           - dangling=<boolean> a volume if referenced or not
+                           - driver=<string> a volume's driver name
+                           - name=<string> a volume's name
       --help               Print usage
       -q, --quiet          Only display volume names
 
-Lists all the volumes Docker knows about. You can filter using the `-f` or `--filter` flag. The filtering format is a `key=value` pair. To specify more than one filter,  pass multiple flags (for example,  `--filter "foo=bar" --filter "bif=baz"`)
```

Comment:
I think we can keep this paragraph, because it also explains the command in general ("Lists all the volumes..."). But you can add something like `. Refer to the [filtering](#filtering) section for more information about available filter options.`